happy/HappyInternet.py

```diff
@@ -445,7 +445,6 @@ def run(self):
                     self.__create_isp()
                     self.__connect_internet_to_isp()
                     self.__ctrl_isp_internet_interface()
-                    self.__nmconf()
```

Comment:
Why did we want to call this before? Why don't we want to call it any longer?

Reply:
We still want it.  Please put it back in.

Reply:
the detail explain is here https://jira.nestlabs.com/browse/CIEC-3087
it seems not correct step now base on my debugging, I don't know why we had this before.
@robszewczyk do you know why we had this before?

Reply:
I have changed back for this one.